repeater/src/main/scala/com.snowplowanalytics.snowplow.storage.bigquery.repeater/Flow.scala

```diff
@@ -42,48 +41,66 @@ object Flow {
     * @param resources all application resources
     * @param events stream of events from PubSub
     */
-  def sink[F[_]: Logger: Timer: Concurrent: ContextShift](resources: Resources[F])(events: EventStream[F]): Stream[F, Unit] =
+  def sink[F[_]: Logger: Timer: Concurrent: ContextShift](
+    resources: Resources[F]
+  )(events: EventStream[F]): Stream[F, Unit] =
     events.parEvalMapUnordered(resources.concurrency) { event =>
-      val insert = checkAndInsert[F](resources.bigQuery, resources.env.config.datasetId, resources.env.config.tableId, resources.backoffTime)(event)
+      val insert = checkAndInsert[F](
+        resources.bigQuery,
+        resources.env.config.datasetId,
+        resources.env.config.tableId,
+        resources.backoffTime
+      )(event)
```

Comment:
Did you change scalafmt in this commit 7b4e82dbf90dad9156770c0857ba7424ba5e6e90 ? It seems that a lot of changes are code formatting, it's hard to see what exactly is related to mention life span in statistics logs

Reply:
I changed scalafmt in a later commit but then had to rebase this one and IntelliJ was already set up to format it with the new settings. It made further rebases such a hassle... 

I will definitely think twice about mixing formatting changes with other commits in future.

Reply:
I think that it's a good practice to have anything related to formatting as a very first commit or very last commit of a release, to avoid any problems when rebasing (it was such a pain when rebasing R118).